lib/linguist/heuristics.yml

```diff
@@ -193,7 +193,9 @@ disambiguations:
   - language: GLSL
     pattern: '^#version\s+[0-9]+\b'
   - language: Gosu
-    pattern: '^uses java\.'
+    pattern: '^uses (java|gw)\.'
```

Comment:
It appears this syntax is legal too (going by [`samples/Genie/web.gs`](https://github.com/github/linguist/blob/1c8a7ef6fe67d3290622eb94d9d1f1a51c522e35/samples/Genie/web.gs#L4-L6):

~~~genie
uses
    Gtk
    WebKit
~~~

So I'd use `\s+` instead of a space:

~~~diff
-pattern: '^uses (java|gw)\.'
+pattern: '^uses\s+(java|gw)\.'
~~~

Reply:
It might be confusing, but that line corresponds to the `Gosu` heuristic, not the `Genie` one.

Reply:
Fair enough. LGTM then. 👍